src/ORM/Behavior/TreeBehavior.php

```diff
@@ -76,6 +78,15 @@ class TreeBehavior extends Behavior
     ];
 
     /**
+     * {@inheritDoc}
+     */
+    public function initialize(array $config)
+    {
+        $this->_config['leftField'] = new IdentifierExpression($this->_config['left']);
+        $this->_config['rightField'] = new IdentifierExpression($this->_config['right']);
```

Comment:
`IdentiferExpression` does not care about configuration, it is coded as a no-op when no quoting is enabled